<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_post_ronda</class>
 <widget class="QWidget" name="ventana_post_ronda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>294</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>128</red>
        <green>64</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>128</red>
        <green>64</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>128</red>
        <green>64</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>128</red>
        <green>64</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="titulo">
     <property name="font">
      <font>
       <family>Marker Felt</family>
       <pointsize>36</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>RESUMEN RONDA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="perdidos">
       <property name="text">
        <string>CLIENTES PERDIDOS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="perdidos_lcd"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="atendidos">
       <property name="text">
        <string>CLIENTES ATENDIDOS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="atendidos_lcd"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="dinero">
       <property name="text">
        <string>DINERO ACUMULADO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="dinero_lcd"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="reputacion">
       <property name="text">
        <string>REPUTACIÃ“N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reputacion_valor">
       <property name="text">
        <string>/5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="boton_salir">
       <property name="text">
        <string>SALIR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_guardar">
       <property name="text">
        <string>GUARDAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_continuar">
       <property name="text">
        <string>CONTINUAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
